Tools/TransformTool/TransformTool.cs

```diff
@@ -363,6 +363,8 @@ public void ProcessInput(ActionMapInput input, Action<InputControl> consumeContr
 #if UNITY_EDITOR
 			if (m_PivotRotation == PivotRotation.Local) // Manipulator does not rotate when in global mode
 				manipulatorTransform.rotation = Quaternion.Slerp(manipulatorTransform.rotation, m_TargetRotation, kLazyFollowRotate * deltaTime);
+
+			UnityEditor.Undo.RecordObjects(Selection.transforms, "EditorVR Manipulator");
```

Comment:
Also re: "Move," I wasn't sure if we want to differentiate our actions from regular editor actions.

I was hoping to be able to read this group name when processing undo actions, but it doesn't seem to come through.